crawl-ref/source/spl-miscast.h

```diff
@@ -10,50 +10,58 @@
 #include "mpr.h"
 #include "spl-util.h"
 
-enum nothing_happens_when_type
+enum class nothing_happens
 {
-    NH_DEFAULT,
-    NH_NEVER,
-    NH_ALWAYS,
+    DEFAULT,
+    NEVER,
```

Comment:
Would using lowercase `default` here cause problems (since `default` is a keyword in C++ for use with `switch` statements)?

Reply:
Yes, `default` is a reserved keyword and cannot be used as an identifier.